app/models/post.rb

```diff
@@ -2,4 +2,14 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :comments
   has_many :likes
+
+  after_save :posts_counter
+
+  def posts_counter
+    user.update(post_counter: user.posts.count)
+  end
```

Comment:
- [ ] Seeing as the ERD diagram specifies the column name to be `author_id` instead of `user_id`, the `belongs_to` relationship between the `post` and `author` should be implemented like this 👇 
```ruby
belongs_to :author, class_name: 'User'
```
You will have to rename the `user_id` column on your `posts` table to `author_id` in order to correctly follow the specifications on the ERD diagram